src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -23,40 +23,27 @@ import (
 	"github.com/getlantern/wfilter"
 )
 
+const (
+	logTimestampFormat = "Jan 02 15:04:05.000"
+)
+
 var (
 	log = golog.LoggerFor("flashlight")
 
-	LogTimestampFormat = "Jan 02 15:04:05.000"
-	lang               string
-	tz                 string
-	versionToLoggly    string
-	cfgMutex           sync.Mutex
+	logFile  *rotator.SizeRotator
+	cfgMutex sync.Mutex
 
 	// logglyToken is populated at build time by crosscompile.bash. During
 	// development time, logglyToken will be empty and we won't log to Loggly.
 	logglyToken string
 
-	// A wrapper of the loggly client for the current session.
-	logglyClient *logglyErrorWriter
+	errorOut io.Writer
+	debugOut io.Writer
+
+	lastAddr string
 )
 
 func init() {
```

Comment:
All the basic logging stuff can be done without any external data, so I'm doing it in init() now.